inst/qml/Mimic.qml

```diff
@@ -87,53 +116,66 @@ Form
                     }
                 }
             }
-        }
-    }
-
-	Section {
-        text: qsTr("Plots")
-        CheckBox { 
-            text:   qsTr("Model plot")
-            name:   "pathPlot"
-            id:     pathPlot 
-            CheckBox { text: qsTr("Parameter estimates") ; name: "pathPlotParameter" }
-            CheckBox { text: qsTr("Legend") ; name: "pathPlotLegend" }
-        }
-    }
-    
-	Section {
-        text: qsTr("Advanced")
-        GroupBox {
-            Layout.fillWidth: true
-            RadioButtonGroup {
-                title: qsTr("Missing value handling")
-                name: "naAction"
-                RadioButton { text: qsTr("Full Information Maximum Likelihood") ; name: "fiml" ; checked: true }
-                RadioButton { text: qsTr("Exclude cases listwise")              ; name: "listwise"             }
-            }
-            RadioButtonGroup {
-                title: qsTr("Emulation")
-                name: "emulation"
-                RadioButton { text: qsTr("None")  ; name: "lavaan"  ; checked: true }
-                RadioButton { text: qsTr("Mplus") ; name: "mplus" }
-                RadioButton { text: qsTr("EQS")   ; name: "eqs"   }
+            CIField {
+                text: qsTr("Confidence intervals")
+                name: "ciLevel"
             }
         }
+        
         GroupBox {
             Layout.fillWidth: true
+            CheckBox{name: "standardizedVariable"; label: qsTr("Standardize variables before estimation"); checked: false}
+            DropDown
+			{
+				name: "estimator"
+				id: estimator
+				label: qsTr("Estimator")
+				values: [
+                    { value: "default", label: qsTr("Auto"), checked: true    },
+                    { value: "ml",      label: qsTr("ML")       },
+                    { value: "gls",     label: qsTr("GLS")      },
+                    { value: "wls",     label: qsTr("WLS")      },
+                    { value: "uls",     label: qsTr("ULS")      },
+                    { value: "dwls",    label: qsTr("DWLS")     },
+                    { value: "mlf",     label: qsTr("MLF")      },
+                    { value: "mlr",     label: qsTr("MLR")      }
+                ]
+			}
+            DropDown
```

Comment:
In general it would be good to move the common QML parts into reusable components so that we can reuse the code in all analyses... Could you do that? Let me know if you are not sure how to do that

Reply:
Okay! I'll peek into another analysis that uses reusable components!